O jogo crasha ao instalar a tradução

ele da esse erro aqui, aalguem pode me ajudar?
o jogo simplesmente fecha com a tradução... quando eu coloco os arquivos originais o jogo abre normalmente!

java.lang.RuntimeException: Exception thrown during execution of asynchronous task!
	at com.badlogic.gdx.concurrent.AsyncTask.await(AsyncTask.java:105)
	at com.robotality.pulp.game.screens.AsyncLoaderScreen.await(AsyncLoaderScreen.java:186)
	at com.robotality.pulp.game.screens.AssetLoaderScreen.update(AssetLoaderScreen.java:165)
	at com.robotality.game.screens.GameScreenAdapter.update(GameScreenAdapter.java:134)
	at com.robotality.pulp.PulpHeroAdapter.update(PulpHeroAdapter.java:92)
	at com.robotality.game.loop.FixedTimeStepGameLoop.render(FixedTimeStepGameLoop.java:48)
	at com.robotality.game.adapter.GameApplicationAdapter.render(GameApplicationAdapter.java:151)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Window.update(Lwjgl3Window.java:335)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.renderThreadFunction(Lwjgl3Application.java:205)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.util.concurrent.ExecutionException: com.badlogic.gdx.utils.GdxRuntimeException: No master language code found: -405211242
	at java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.util.concurrent.FutureTask.get(FutureTask.java:192)
	at com.badlogic.gdx.concurrent.AsyncTask.await(AsyncTask.java:90)
	... 9 more
Caused by: com.badlogic.gdx.utils.GdxRuntimeException: No master language code found: -405211242
	at com.robotality.game.i18n.LanguageTable.onJsonRead(LanguageTable.java:79)
	at com.robotality.game.i18n.LanguageTable.refresh(LanguageTable.java:107)
	at com.robotality.game.i18n.LanguageTable.read(LanguageTable.java:268)
	at com.robotality.pulp.i18n.I18N.reload(I18N.java:59)
	at com.robotality.pulp.i18n.I18N.<init>(I18N.java:32)
	at com.robotality.pulp.game.PulpHero.lambda$initializeGameAssets$1(PulpHero.java:334)
	at com.robotality.game.util.DeferredRunnable.execute(DeferredRunnable.java:35)
	at com.robotality.game.util.TimedRunnable.execute(TimedRunnable.java:33)
	at com.robotality.game.util.TimedRunnable.<init>(TimedRunnable.java:21)
	at com.robotality.pulp.game.PulpHero.initializeGameAssets(PulpHero.java:333)
	at com.robotality.pulp.game.screens.AssetLoaderScreen$Job.load(AssetLoaderScreen.java:57)
	at com.robotality.pulp.game.screens.AsyncLoaderScreen$Job.call(AsyncLoaderScreen.java:199)
	at com.robotality.pulp.game.screens.AsyncLoaderScreen$Job.call(AsyncLoaderScreen.java:195)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	... 1 more

--- a/pt-BR/_config.json
+++ b/pt-BR/_config.json
@@ -1,6 +1,0 @@
-{
-"code": "pt-BR",
-"systemCode": "pt",
-"asianLanguage": false,
-"label": "Português Brazil"
-}

